briefcase/app.py

```diff
@@ -52,6 +52,14 @@ class app(Command):
          'URL for the support package to use'),
         ('download-dir=', None,
          "Directory where the project support packages will be cached"),
+        ('build', None,
+         "Build the project after generating"),
+        ('execute', None,
+         "Run the application after building"),
+        ('os-version=', None,
+         "For iOS, the OS version to run (e.g., iOS 10.2)"),
```

Comment:
Let's not make the docstring iOS-specific. What if we wanted to target a specific android version some day?

Maybe make the docstring something like "Sets the OS version. Currently only iOS supported."